sdk/python/kfp/compiler/compiler_test.py

```diff
@@ -292,9 +292,9 @@ def test_passing_string_parameter_to_artifact_should_error(self):
       """)
         with self.assertRaisesRegex(
                 type_utils.InconsistentTypeException,
-                'Incompatible argument passed to the input "some_input" of '
-                'component "compoent": Argument type "STRING" is incompatible '
-                'with the input type "system.Artifact@0.0.1"'):
+                "Incompatible argument passed to the input 'some_input' of "
+                "component 'component': Argument type 'STRING' is incompatible "
+                "with the input type 'system.Artifact@0.0.1'"):
```

Comment:
Normalized strings in type checking; this is the updated test case

Reply:
Hmm, not sure if this is needed or worth doing. When the message is printed out, it would show the inner quotes only, which is `"component"` before the change. 
And double-quotes are the universally accepted "string quotes", while single-quotes are only valid in certain languages, Python, Javascript for example, but not C/C++, Java, Golang, etc.